packages/vector_graphics/lib/src/render_vector_graphics.dart

```diff
@@ -31,12 +40,40 @@ class _RasterKey {
   int get hashCode => Object.hash(assetKey, width, height);
 }
 
-class _RasterData {
-  _RasterData(this.image, this.count, this.key);
+/// The cache entry for a rasterized vector graphic.
+class RasterData {
+  /// Create a new [RasterData].
+  RasterData(this.image, this.count, this.key);
 
+  /// The rasterized vector graphic.
   final ui.Image image;
-  final _RasterKey key;
+
+  /// The cache key used to identify this vector graphic.
+  final RasterKey key;
+
+  /// The number of render objects currently using this
+  /// vector graphic raster data.
   int count = 0;
+  bool _disposed = false;
+
+  /// Dispose this raster data, including the attached image.
+  ///
+  /// Will debug print if disposed more than once.
+  void dispose([RenderObject? object]) {
```

Comment:
We don't have any errors show us using a disposed image though, just calling dispose twice. This is mostly just for me to debug, since I can't run the test locally